README.md

```diff
@@ -28,36 +28,9 @@ Git on Windows will, by default, automatically convert line endings from `lf` to
 
 See [this writeup](https://help.github.com/articles/dealing-with-line-endings/) for more information.
 
-2. Create the TFB Docker virtual network
+2. Run a test.
 
-        $ docker network create tfb
-
-3. Run a test.
-
-        $ docker run -it --network=tfb -v /var/run/docker.sock:/var/run/docker.sock --mount type=bind,source=[ABS PATH TO THIS DIR],target=/FrameworkBenchmarks techempower/tfb --mode verify --test gemini
```

Comment:
Also, since this doesn't work on Windows at all, we need to keep the `docker run` command in the instructions.

Reply:
OK, makes sense. I'll be less heavy-handed on the readme :)

The opther option of course is to always direct windows users at vagrant.